src/stcal/jump/twopoint_difference.py

```diff
@@ -54,6 +59,26 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
         neighbors (marginal detections). Any primary jump below this value will
         not have its neighbors flagged.
 
+    dqflags: dict
+        A dictionary with at least the following keywords:
+        DO_NOT_USE, SATURATED, JUMP_DET, NO_GAIN_VALUE, GOOD
+
+    after_jump_flag_e1 : float
```

Comment:
the 'min/max_jump_to_flag_neighbors' flags are in units of sigma, does it make more sense to express these new parameters in units of electrons as they currently are, or would sigma be suitable? just wanted to check because i could see this getting potentially confusing with different units, but if it makes more sense based on the nature of the issue then thats fine

Reply:
The after jump transient effect can change based on the amplitude of the ramp jump, not the ratio to the noise.  Hence we need the actual ramp jump amplitude.